lib/sse_test_server/sse_server.ex

```diff
@@ -0,0 +1,137 @@
+defmodule SSETestServer.SSEServer do
+  @moduledoc """
+  An HTTP server for testing SSE clients.
+
+  TODO: Document properly.
```

Comment:
I really think this needs just a little more documentation... Like, I'm not clear why the SSEServer has multiple endpoints, or how events are sent to the different endpoints (I _think_ an event is specific to an endpoint?).

Just a small usage example would go a long way.

Reply:
Ok I think the tests are probably the best documentation for this currently. I'm somehow not in the habit of reading those first and now understand this better.

Reply:
Yes, it does need documentation here. I'll add some.